src/components/usa-alert/usa-alert.stories.js

```diff
@@ -0,0 +1,60 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-alert.twig";
+import {
+  Data,
+  EmergencyData,
+  ErrorData,
+  InfoData,
+  NoHeaderData,
+  NoIconData,
+  SlimData,
+  SuccessData,
+  WarningData,
+} from "./content";
+
+export default {
+  title: "Components/Alert",
+  argTypes: {
+    modifier: {
+      control: "text",
+    },
+    role: {
+      control: "text",
+    },
+    title: {
+      control: "text",
+    },
+    text: {
+      control: "text",
+    },
+  },
```

Comment:
We should be able to safely remove all of these argTypes from stories.js since they are already defined in the JSON files. This will allow storybook to show only the relevant controls for the story instead of showing empty values